src/app/workflow/launch-third-party/multi-cloud-launch/multi-cloud-launch.component.ts

```diff
@@ -116,27 +117,29 @@ export class MultiCloudLaunchComponent extends Base implements OnInit {
       localStorage.setItem('useCustomLaunch', 'false');
     }
 
-    // Uncomment when we want to create and save a user's custom launch entry
-    // if (this.presetLaunchWithOption === 'other' && this.user) {
-    //   const url: URL = new URL(this.launchWith.url);
-    //   const newCustomInstance: CloudInstance = {
-    //     url: this.launchWith.url,
-    //     partner: this.languagePartner,
-    //     supportsFileImports: null,
-    //     supportsHttpImports: null,
-    //     supportedLanguages: new Array<Language>(),
-    //     displayName: url.hostname,
-    //   };
-    //
-    //   this.usersService.postUserCloudInstance(this.user.id, newCustomInstance).subscribe(
-    //     (usersCloudInstances: Array<CloudInstance>) => {
-    //       this.usersCloudInstances = usersCloudInstances;
-    //     },
-    //     (error: HttpErrorResponse) => {
-    //       console.log(error.message);
-    //     }
-    //   );
-    // }
+    // Create and save a user's custom launch entry to the db
+    if (this.presetLaunchWithOption === 'other' && this.user) {
+      const url: URL = new URL(this.launchWith.url);
```

Comment:
This will throw an exception if the URL is invalid, e.g., `usegalaxy.org` instead of `https://usegalaxy.org`, but based on your comments below, it doesn't seem like it matters at this point.